sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -464,7 +464,18 @@ tuple val(meta), path('*.tab'), emit: tab,  optional: true
 
 ### One output channel per output file type
 
-Each output file SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+Each output file type SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+
+:::info{title="Rationale" collapse}
+The instruction to emit each output file type in its own channel ensures better organization and manageability of the output files. By assigning a dedicated channel to each file type, you can prevent clutter and potential confusion that might arise from mixing different file types in a single channel. This approach also simplifies the process of retrieving and processing specific types of output, as each type can be easily identified and accessed within its designated channel.
+
+In some cases the file format can be different between files of the same type (e.g. indices: `.bai` and `.crai`). These different file formats SHOULD be part of the same output channel since they are the same file type.
```

Comment:
Imo this should be outside of the rationale block, because it contains guidelines